src/metabase/server/middleware/session.clj

```diff
@@ -232,21 +231,20 @@
    (fn [db-type max-age-minutes session-type enable-advanced-permissions?]
      (first
       (db/honeysql->sql
-       (cond->
-         {:select    [[:session.user_id :metabase-user-id]
-                      [:user.is_superuser :is-superuser?]
-                      [:user.locale :user-locale]]
-          :from      [[Session :session]]
```

Comment:
Using a model in a Honey SQL query doesn't work in Honey SQL 2 so this change will set us up for the switch

Reply:
Is there a Toucan 2-friendly way to go from the Toucan 2 model keyword to the actual table? I'm imagining I'm familiar with the model keyword (`:dashboard` or `:card`) and don't want to have to remember the legacy table names (`:report_dashboard` and `:report_card`).

Then again, if I'm using HoneySQL directly, we're already deep into the table structure.